src/contexts/bridge.context.tsx

```diff
@@ -34,7 +60,45 @@ const BridgeProvider: FC = (props) => {
     [env]
   );
 
-  return <bridgeContext.Provider value={{ getBridges }} {...props} />;
+  const getClaimStatus = useCallback(
+    ({ originNetwork, depositCount }: GetClaimStatusParams) => {
+      if (env === undefined) {
+        throw new Error("The bridge API client couldn't be instantiated");
```

Comment:
I think `instantiated` is a misleading word since we're not instantiating any class here.

Also, I'd rather be specific about the source of the error being the Env:

> "The bridge API can not be used. Env is not ready"

Or just:

> "Env is not ready"

Since the consumer of this function already knows that it is calling the Bridge's API function `getClaimStatus`.

Reply:
Right! Updated